aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -586,6 +604,36 @@ module aptos_token::token {
         direct_deposit(to, token);
     }
 
+
+    /// Token owner can create this one-time withdraw capability with an expiration time
+    public fun create_withdraw_capability(
```

Comment:
Hmm what if someone issues two different withdraw capabilities? Wouldn't this lead to tug of war?

Reply:
no war only first come first serve if token is limited

Reply:
I see - I missed the token_owner field. With that said though, in general, is it safe to allow issuing multiple WithdrawCapability objects?

Reply:
There are edge cases for example on amount as well. Owner can also use WithdrawCap to double or triple sell their tokens

Reply:
It is common that people listing their token on multiple places for sale.